monix-java/src/main/scala/monix/java8/execution/package.scala

```diff
@@ -65,7 +64,6 @@ package object execution {
     * DEPRECATED — switch to Scala 2.12+ and
     * [[monix.execution.FutureUtils.toJavaCompletable FutureUtils.toJavaCompletable]].
     */
-  @deprecated("Switch to Scala 2.12+ and FutureUtils.toJavaCompletable", "3.0.0")
```

Comment:
same

Reply:
Same as the others — Scala 3 generates warnings, even if the extensions don't get used, which is why I prefer to have the deprecation warnings only on the methods themselves.